src/events/SDL_dragevents.c

```diff
@@ -0,0 +1,64 @@
+#include "../SDL_internal.h"
+
+#include "SDL_events.h"
+#include "SDL_events_c.h"
+#include "SDL_dragevents_c.h"
+
+#include "../video/SDL_sysvideo.h"  /* for SDL_Window internals. */
+
+const int
+SDL_SendDrag(SDL_Window *window, const SDL_EventType evtype, const char *data, int x, int y)
```

Comment:
Nice! So this is the 'platform-agnostic' piece, and what we would call into if we implemented this on Windows / Linux?

Reply:
Yep -- if you want a reference check out `src/Events/SDL_dropevents.c`. I modeled it very closely after that!